doc/source/tutorial/stats/continuous_geninvgauss.rst

```diff
@@ -0,0 +1,20 @@
+
+.. _continuous-geninvgauss:
+
+Generalized Inverse Gaussian Distribution
+=========================================
+
+The probability density function is given by:
+
+.. math::
+	:nowrap:
+	
+	\begin{eqnarray*}
+	        f(x; p, b) = x^{p-1} \exp(-b(x + 1/x)/2) / (2 K_p(b)),
+	\end{eqnarray*}
+
+where :math:`x > 0` is a real number and the parameters :math:`p, b` satisfy :math:`b > 0`. :math:`K_v` is the modified Bessel function of second kind of order :math:`p` (`scipy.special.kv`).
```

Comment:
Should K_v  be K_p, as in ":math:`K_p`"?

Reply:
yes, i changed :math:`p` to :math:`v` to have better alignment to `Special.kv`